packages/flutter/test/material/data_table_test.dart

```diff
@@ -627,14 +627,16 @@ void main() {
     Widget buildCustomTable({
       int? sortColumnIndex,
       bool sortAscending = true,
-      double dataRowHeight = 48.0,
```

Comment:
Can we maintain the original tests? While it is deprecated, we should still ensure that the original property works while it is deprecated.

Reply:
Yes, there is a separate test for dataRowHeight. see line 780 in the changed data_table_test.dart file.